.github/workflows/deploy.yaml

```
@@ -387,6 +387,16 @@ jobs:
               echo "No .yml file found in ${TARGET_DIR}."
               exit 1
           fi
+
+          # Read team and division from catalog-info.yaml
+          backstage_owner=SRE
+          backstage_system=Platform
+          if [ -e ./catalog-info.yaml ]; then
+            # Extract the 'owner' and 'system' values, removing anything after a '#'
+            OWNER=$(grep '^\s*owner:' ./catalog-info.yaml | sed 's/.*owner: \([^#]*\).*/\1/' | xargs)
```

could we add optional `?` on the whitespace after the colon? Both for owner and system parsing
```suggestion
            OWNER=$(grep '^\s*owner:' ./catalog-info.yaml | sed 's/.*owner: ?\([^#]*\).*/\1/' | xargs)
```